Silicon/ElkhartlakePkg/ElkhartlakePkg.dec

```diff
@@ -0,0 +1,26 @@
+#
+# This file contains an 'Intel Peripheral Driver' and is
+# licensed for Intel CPUs and chipsets under the terms of your
+# license agreement with Intel or your vendor.  This file may
+# be modified by the user, subject to additional terms of the
+# license agreement

```

Comment:
Should this be removed since BSD-2 license is used below?

Reply:
Done removed.